### DISCUSSION TITLE
Glew documentation

### DISCUSSION THREAD

**Discussion by Ab0Amin** [nigels-com/glew#287] [Q&A] [answered]
Hi wish all's good,
I currently start using Glew in my app and searching for documentation everywhere and can't find it.
I'll be thanks fall if u help me to find it.
thanks in advance.

**Comment by nigels-com** [comment, marked as answer]
Currently, three main pages:
- [README.md](https://github.com/nigels-com/glew/blob/master/README.md)
- [Basic Usage](http://glew.sourceforge.net/basic.html)
- [Code Generation](http://glew.sourceforge.net/advanced.html)

For general OpenGL documentation:
- [opengl.org](https://www.khronos.org/registry/OpenGL/index_gl.php)
- [OpenGL Programming Guide: The Official Guide to Learning OpenGL](https://www.amazon.com.au/OpenGL-Programming-Guide-Official-Learning/dp/0134495497)

Hope it helps.

**Reply by Ab0Amin** [reply]
but there are no other documentations describe The classes and functions with some examples?
if not how should I know the behavior of the function or which function that I need.
thanks in advance.

**Reply by nigels-com** [reply]
Added recommended links for OpenGL documentation.
